convert2rhel/main.py

```diff
@@ -298,3 +302,24 @@ def rollback_changes():
             raise
 
     return
+
+
+def print_data_collection():
+    """Print information about telemetry and ask for acknowledge."""
+    if not "CONVERT2RHEL_DISABLE_TELEMETRY" in os.environ:
+        loggerinst.info(
+            "The convert2rhel utility uploads the following data about the system conversion"
+            " to Red Hat servers for the purpose of the utility usage analysis:\n"
+            "- The Conver2RHEL command as executed\n"
```

Comment:
I think I made a typo in my last suggestion here, sorry: 

```suggestion
            "- The Convert2RHEL command as executed\n"
```